exportScriptForLinux/createBackupForLinux.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+H2_VERSION=1.4.197
+DB_URL=jdbc:h2:file:./db/$H2_VERSION/keeptime-h2-db;DB_CLOSE_ON_EXIT=FALSE
+DB_USER=sa
+DB_PASSWD=
+SQL_BACKUP_FILENAME=KeepTime_database-export_H2-version-$H2_VERSION.sql
+
+echo Get H2 jar from KeepTime jar ....
+jar xf keeptime-1.2.0-SNAPSHOT.jar
+rm -r org 
+rm -r META-INF
+echo 
+
+DRIVER_JAR_FROM_KEEPTIME=BOOT-INF/lib/h2-$H2_VERSION.jar
+
+echo Filename : $SQL_BACKUP_FILENAME
+echo 
+echo Location : 
+pwd  
+echo 
+
+     
+	java -cp $DRIVER_JAR_FROM_KEEPTIME org.h2.tools.Script -url $DB_URL -user $DB_USER -script $SQL_BACKUP_FILENAME -options DROP
+rm -r BOOT-INF
+
+echo delete Files from unzipping KeepTime
+echo Export succeeded 
```

Comment:
Works. I did clean it up a bit, also only extracting the needed h2.jar file. Added some more info for the user as well:

```suggestion
#!/bin/bash
H2_VERSION=1.4.197
DB_URL=jdbc:h2:file:./db/$H2_VERSION/keeptime-h2-db;DB_CLOSE_ON_EXIT=FALSE
DB_USER=sa
DB_PASSWD=
SQL_BACKUP_FILENAME=KeepTime_database-export_H2-version-$H2_VERSION.sql

echo "Script to export the content of H2 databse from KeepTime version 1.2.0-SNAPSHOT to .sql file."
echo "The script should be placed next to the .jar file."
read -p "Press enter to continue"

echo -e "\nExtracting H2 driver from KeepTime .jar file."
DRIVER_JAR_FROM_KEEPTIME=BOOT-INF/lib/h2-$H2_VERSION.jar
jar xfv keeptime-1.2.0-SNAPSHOT.jar $DRIVER_JAR_FROM_KEEPTIME

echo -e "\nTriggering database backup."
java -cp $DRIVER_JAR_FROM_KEEPTIME org.h2.tools.Script -url $DB_URL -user $DB_USER -script $SQL_BACKUP_FILENAME -options DROP

echo -e "\nCleanup of extracted .jar file."
rm -rv BOOT-INF

echo -e "\nIf no error occured the export has succeeded."
echo "The exported file was created at '$(pwd)/$SQL_BACKUP_FILENAME'."
echo -e "\nYou can now import this file in a new version of KeepTime.\n"

read -p "All set. Press enter to exit script."
```
![image](https://user-images.githubusercontent.com/5772511/214064275-3f4fd131-98ba-4443-80f4-0f6270d411fd.png)

@DavidDamke could you revalidate the changes I made?